utils/stoken/Makefile

```diff
@@ -20,7 +20,7 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_REV).tar.gz
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 PKG_MAINTAINER:=Florian Fainelli <florian@openwrt.org>
 PKG_LICENSE:=LGPL-2.1
-PKG_INSTALL:=1
+PKG_INSTALL:=2
```

Comment:
Did you mean to change PKG_VERSION here?  PKG_INSTALL is the flag to run the "make install" step